docker-compose.yml

```diff
@@ -6,24 +6,25 @@ services:
     environment:
       PROVISION_DIRECTORIES: "1000:1000:0755:/tmp/certs"
     volumes:
-    - "./certs:/tmp/certs"
+      - "./certs:/tmp/certs"
```

Comment:
Should this not be “./certs:/certs” ?

Reply:
no, this one is fine, local directory `certs` will be mounted as `/tmp/certs` to container, value is correct in `PROVISION_DIRECTORIES`, there is no need to change it

Reply:
ah, yes. `"1000:1000:0755:/tmp/certs"` 👍